ui/js/models/resultset.js

```diff
@@ -267,7 +267,12 @@ treeherder.factory('ThResultSetModel', ['$http', '$location',
                     const graph = resp.data;
 
                     // Build a mapping of buildbot buildername to taskcluster tasklabel for bbb tasks
-                    const builderToTask = _.omit(_.invert(_.mapValues(graph, 'task.payload.buildername')), [undefined]);
+                    const builderToTask = _.omit(_.invert(Object.entries(graph)
```

Comment:
I think your best bet to track this down would be to test it and see what error message you get.  But this function retriggers jobs, and it's best not to do that unless you know it's needed.  So you can cut off the action in this function by adding a return on line 276, or thereabouts.
I would also re-add the old function to make the ``const`` of ``builderToTaskOld`` or something.  Then after your new function, print both the old and new to the console to be sure you're getting the same result.

To run this locally, do ``yarn start:stage``

Then, one it's running, open your development console and log in/register.  Then you click the down arrow at the top of one of the pushes and select "Add new jobs":

![screenshot 2018-07-18 09 18 09](https://user-images.githubusercontent.com/419924/42894412-a9857102-8a6b-11e8-913a-591af801e1cf.png)

That will add a bunch of yellow-gray "run-able jobs".  Click one of those, then click ``Trigger new jobs``.  This will invoke the function you're modifying

![screenshot 2018-07-18 09 20 49](https://user-images.githubusercontent.com/419924/42894536-ee14e9f6-8a6b-11e8-9891-caeb6af24c49.png)

Best of luck tracking down the problem!  :)

Reply:
@camd Thanks for the detailed explanation. I will try those steps and get back to you, soon!